crit/Makefile

```diff
@@ -37,9 +38,24 @@ gen-proto:
 		$(proto_files)
 
 bin/crit: cmd/cli.go
-	$(GO) build -o $@ $^
+	$(GO) build ${GOFLAGS} -o $@ $^
+
+../test/loop/loop:
+	$(MAKE) -C ../test/loop
+
+test-imgs: ../test/loop/loop
+	$(eval PID := $(shell ../test/loop/loop))
+	mkdir -p $@
+	$(CRIU) dump -v4 -o dump.log -D $@ -t $(PID)
+	$(CRIU) restore -v4 -o restore.log -D $@ -d
+	pkill -9 loop
+
+unit-test: test-imgs
+	$(eval GOFLAGS ?= -cover)
```

Comment:
How does `eval` work? That is new to me.

Reply:
> it allows you to define new makefile constructs that are not constant; which are the result of evaluating other variables and functions.

https://www.gnu.org/software/make/manual/html_node/Eval-Function.html

Reply:
`GOFLAGS` will be defined by default only in `unit-test`, but not in `bin/crit`.